src/features/emojiMod.ts

```diff
@@ -69,6 +69,24 @@ export const reactionHandlers: ReactionHandlers = {
 
     reportUser({ reason, message, staff, members });
   },
+  "🔍": async ({ message, usersWhoReacted }) => {
+    const NUMBER_OF_REACTIONS_REQUIRED_FOR_DELETION = 2;
+
+    const hasRequiredNumberOfReactions =
+      usersWhoReacted.length >= NUMBER_OF_REACTIONS_REQUIRED_FOR_DELETION;
+
+    const hasAuthoritativeUserReacted = usersWhoReacted.some(
+      (userWhoReacted) => isStaff(userWhoReacted) || isHelpful(userWhoReacted),
+    );
+
+    if (!hasRequiredNumberOfReactions || !hasAuthoritativeUserReacted) {
+      return;
+    }
+
+    await message.delete();
+
+    await message.author.send("...");
```

Comment:
I'm leaving the blurb up for debate. What message do we want to send to the author of the message?

Reply:
Do you think it would be feasible to set this up to reply in a thread, instead of DMing the author? I think DMs are easier to miss, and more people have them disabled (which would mean the author gets no indication for why their message was removed). That adds more complexity, like what happens if the message is already in a thread, but I think it's a better experience

Reply:
I agree with the downsides of using a direct message 👍 perhaps sending the message in the channel and mentioning the author would solve this and also act as a deterrent and provide passive information to other users? I think a thread is likely feasible if you feel strongly about it, although I'm keen on removing the messages to really try and strongly enforce a higher quality of question 🙂

Reply:
For a blurb, maybe something like
> Sorry, our most active helpers have flagged this as a question that needs more work before a good answer can be given. This may be because it's ambiguous, too broad, or otherwise challenging to answer. This is a good resource about asking good programming questions: https://zellwk.com/blog/asking-questions/

Reply:
Absolutely perfect, thanks. I'll use that 🎉

Reply:
> I think a thread is likely feasible if you feel strongly about it, although I'm keen on removing the messages to really try and strongly enforce a higher quality of question 🙂
Agreed! Actually I was thinking of threads as an ideal way to approach that, in fact — can open a thread, send a message to it, then remove the initial message, leaving very little trace of both the question and the intervention to an outside observer

Reply:
If this behaves as you expect, I'll guard against reactions within a thread and add the reporting functionality 🎉

Reply:
I've done this 👍